blog.html

```diff
@@ -20,15 +20,15 @@
     <header>
         <div class="btn logged-out" id="out-button">
             <!-- <a href="subscribe.html" target="_blank">Subscribe</a> -->
-            <a href="signup.html">Signup</a>
-            <a href="login.html">Log in</a>
+            <a href="./signup.html">Signup</a>
```

Comment:
Check if all files use the same path convention.
There is a difference between putting the slash first and the dot (.)

Read about the difference between relative and absolute paths